README.md

```diff
@@ -310,19 +315,40 @@ repositories:
     #   equivalent to: git clone https://github.com/CiscoTestAutomation/examples /workspace/examples
     examples:
         url: https://github.com/CiscoTestAutomation/examples
+        ssh_key: <id_rsa file contents>
 
     #   equivalent to: mkdir -p /workspace/solutions; git clone https://github.com/CiscoTestAutomation/examples /workspace/solutions/examples
     solutions/examples:
         url: https://github.com/CiscoTestAutomation/solution_examples
+        credentials:
+            username: <git username>
+            password: <git password>
+
+```
+
+#### `yaml loader`
 
+Host environment variables to be loaded into the build yaml. This provides a way
+to dynamically substitute variables into the yaml without saving them
+in the resulting yaml. This is an effective way to store and save secrets in
```

Comment:
This makes it sound like using `%ENV{}` anywhere in the yaml is kept secret, but only the ssh_key and credentials are removed from the build yaml.

Reply:
removed the comments about secrets. I just provides a way to do dynamic substitution. Got confused because it does this for us in xpresso since we store the build.yaml on the server so we dont see it because its %ENV{}.